eclipse/launchers/LobbyRunner.launch

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <launchConfiguration type="org.eclipse.jdt.launching.localJavaApplication">
 <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_PATHS">
-<listEntry value="/game-core/src/main/java/games/strategy/engine/lobby/server/LobbyServer.java"/>
+<listEntry value="/lobby/src/main/java/games/strategy/engine/lobby/server/LobbyRunner.java"/>
 </listAttribute>
 <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_TYPES">
 <listEntry value="1"/>
 </listAttribute>
-<stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="games.strategy.engine.lobby.server.LobbyServer"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="game-core"/>
+<stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="games.strategy.engine.lobby.server.LobbyRunner"/>
+<stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="lobby"/>
 <stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-ea"/>
+<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:game-core}"/>
```

Comment:
This is a hack to workaround the fact that the path used by `LobbyPropertyReader` is hard-coded to be _config/lobby/lobby.properties_, which exists in the `game-core` subproject.  We might have to revisit this in the future and resolve it with the _lobby.properties_ file in the `lobby` subproject.

Reply:
To confirm, you caught the hack I added by duplicating the properties file? There is actually oen in the lobby project, if we can use that one, it would be ideal.

Reply:
Actually, I take that back, best to confirm. I reverted and rolled forward enough times I lost track : (   The project split was largely to enable flyway as a next move and get the boat turning in this direction. 

Thank you for helping with the migration, tying up loose ends :100: